cmd/ctr/checkpoint.go

```diff
@@ -48,6 +49,19 @@ var taskCheckpointCommand = cli.Command{
 		if err != nil {
 			return err
 		}
+
+		labels := map[string]string{
+			"containerd.io/checkpoint": "true",
```

Comment:
Is this the label convention we are pushing?

@dmcgowan @crosbymichael

Reply:
Yes, filtering on type would be way better.

@wenjianhn Could you try to make this work with the mediatype of the image?

Reply:
Does application/vnd.oci.image.index.v1+json mean the image is a checkpoint?

@crosbymichael @stevvooe

Reply:
No, that means its an OCI index. I suppose we are not exposing the checkpoint at the top-level. The mediatype for the checkpoint will be buried in the `manifests` property of the index.

@crosbymichael Was this what you had in mind?

Reply:
@stevvooe ya that is correct that it should be a manifest list and the entire list represents a "checkpoint"

I'm not sure how we represent this at the image level, i guess labels but they should be adding in the `containerd` client.

Reply:
@crosbymichael I think labeling the content here is sufficient. It provides a way of clients to indicate that something has some capability that might be expensive to calculate. These labels would be scoped to only containerd.

So, I guess, back to the question: is this the format we are using for labels?

Reply:
@stevvooe I think so.  I just saw @dmcgowan GC pr that uses labels like this for GC objects.